### PR TITLE
Support minimal-profile binary size measurement.

### PR DESCRIPTION
**Pull request by rtakacs**

### CODE REVIEW

**Comment by zherczeg** [comment]
Looks good

**Comment by robertsipka** [comment]
LGTM

File: API/application/iotjs.py [hunk 1]
```diff
@@ -111,59 +117,65 @@ def build(self, device):
...
         os.build(self, self.buildtype, build_flags, 'all')
+        utils.copy_file(mapfile, paths.IOTJS_MINIMAL_MAP_FILE_PATH)
 
-        utils.mkdir(paths.IOTJS_MAP_DIR_PATH)
-        utils.copy_file(mapfile, paths.IOTJS_MAP_FILE_PATH)
+        # Step 3: Create target specific profile build for the binary size measurement.
+        build_flags = build_flags[:-1]
+        build_flags.append('--profile=%s' % profile)
+
+        if device.get_type() in ['stm32f4dis', 'rpi2']:
```
Comment (LaszloLango) [inline API/application/iotjs.py]: I'd put `os.prebuild(self)` before this to ensure a clean build.

**Comment by rtakacs** [comment]
@LaszloLango I've just updated the PR.

**Pull request merged**